checador/app/views/employees/employee.html.erb

```diff
@@ -0,0 +1,40 @@
+
+<div class="container mt-5">
+    <P class="fs-1">Employees</P>
+  <div class="float-start">
+    <%= link_to new_employee_path, class: 'btn btn-primary' do %>
+      <i >AGREGAR</i>
+    <% end %>
+  </div>
+  <table class="table table-bordered">
+    <thead class="table-primary">
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Namee</th>
+        <th scope="col">Email</th>
+        <th scope="col">Position</th>
+        <th scope="col">Employee Number</th>
+        <th scope="col">Private Number</th>
+        <th scope="col">Status</th>
+        <th scope="col">Actions</th>
+      </tr>
+    </thead>
+     <tbody>
+        <tr>
+          <td>1</td>
+          <td>Juan</td>
+          <td>juanbonix@gmail.com</td>
+          <td>Gerente</td>
+          <td>2</td>
+          <td>23452</td>
+          <td>Active</td>
+          <td>
+            <%= link_to new_employee_path, class: 'btn btn-primary' do %>
+             <i >EDIT</i>
+                 <% end %>
```

Comment:
You can also do
`<%= link_to "EDIT", new_employee_path, class:"btn btn-primary"%`

Reply:
ok.